lib/lexisnexis/instant_verify/proofer.rb

```diff
@@ -14,7 +14,7 @@ class Proofer < LexisNexis::Proofer
                           :state,
                           :zipcode
 
-      optional_attributes :address2, :uuid_prefix, :dob_year_only
```

Comment:
This attribute will come back in future iterations, but it seems easier to me to just wholesale revert it rather than keep it around and have it appear temporarily unused